Gems/Atom/RHI/Metal/Code/Source/RHI/Device.cpp

```diff
@@ -404,7 +404,7 @@ namespace AZ
             return m_nullDescriptorManager;
         }
         
-        AZStd::vector<RHI::Format> Device::GetValidSwapChainImageFormats(const RHI::WindowHandle& windowHandle) const
+        AZStd::vector<RHI::Format> Device::GetValidSwapChainImageFormats(const RHI::WindowHandle& windowHandle, const NativeConnection& nativeConnection) const
```

Comment:
nit: I would add maybe_unused to the parameters here as well.

Reply:
Use SDL instead of xcb create windows is a better choose, m_nativeConnection is not neeed.